lib/src/sdk_api/view/video_view.dart

```diff
@@ -10,36 +9,49 @@ import 'package:flutter/widgets.dart';
 
 import 'package:dolbyio_comms_sdk_flutter/dolbyio_comms_sdk_flutter.dart';
 
-/// A controller for the [VideoView] that is responsible for attaching a [Participant] and a [MediaStream] to the [VideoView], detaching them, and getting information about the [VideoView] state.
+/// A controller for the [VideoView] that is responsible for attaching a [Participant] and a
+/// [MediaStream] to the [VideoView], detaching them, and getting information about the
+/// [VideoView] state.
 /// 
 /// An instance of this class can be provided during the instantiation of the [VideoView] widget, 
 /// if this widget is constructed using [VideoView.forList].
 class VideoViewController {
 
   _VideoViewState?  _state;
+
+  Participant? _participant;
```

Comment:
participant and mediaStream in controller is unnecessary. This fields are only set in attach method and pass to VideoViewState object.